packages/core/package.json

```diff
@@ -62,6 +62,7 @@
     "build": "theiaext build",
     "watch": "theiaext watch",
     "test": "theiaext test",
+    "test:watch": "theiaext test:watch",
```

Comment:
@kittaakos I find it's strange:
- put a spec test file not next to a tested file
- to declare frontend APIs in common instead of the browser. Although I think that a separation of APIs and implementation is good, but it can be done by a new folder with 2 files under the browser folder:
  - one file with APIs reexported by this folder and by the browser folder
  - another file with the implementation which is used within the browser folder

I think that both APIs and tests should be put in browser, jsdom should be used to mock `window.setTimeout` instead of introducing an abstract superclass then everything can be encapsulated within the browser folder, not to be spread across the core extension.

Other opinions?

Reply:
Yes I agree, as we've already started using `jsdom` in other places where frontend tests would fail because of the node environment, it would make sense to have the `.spec.ts` next to the service being tested.

Reply:
@svenefftinge can you review when you have time?

We need to adjust documentation if we change rules how tests and code should be located.